src/pages/PasswordRecovery.vue

```diff
@@ -39,7 +40,12 @@
         </template>
         <template #second>
           <router-link to="/login">
-            <n-button strong type="primary" class="flex !w-full rounded-md">
+            <n-button
+              strong
+              color="#434190"
```

Comment:
Yesterday I spent some time making the buttons work correctly. We can't hardcode colors, we have to change the theme: https://www.naiveui.com/en-US/os-theme/docs/customize-theme

This will make it consistent across the app without having to change the color of every button we use and it will not be necessary to change the attributes I added yesterday in the code.